Source/automap.cpp

```diff
@@ -504,23 +736,24 @@ void DrawAutomapText(const Surface &out)
 		return;
 	}
 
-	if (leveltype != DTYPE_TOWN) {
-		std::string description;
-		switch (leveltype) {
-		case DTYPE_NEST:
-			description = fmt::format(fmt::runtime(_("Level: Nest {:d}")), currlevel - 16);
-			break;
-		case DTYPE_CRYPT:
-			description = fmt::format(fmt::runtime(_("Level: Crypt {:d}")), currlevel - 20);
-			break;
-		default:
-			description = fmt::format(fmt::runtime(_("Level: {:d}")), currlevel);
-			break;
-		}
-
-		DrawString(out, description, linePosition);
-		linePosition.y += 15;
+	std::string description;
+	switch (leveltype) {
+	case DTYPE_NEST:
+		description = fmt::format(fmt::runtime(_("Level: Nest {:d}")), currlevel - 16);
+		break;
+	case DTYPE_CRYPT:
+		description = fmt::format(fmt::runtime(_("Level: Crypt {:d}")), currlevel - 20);
+		break;
+	case DTYPE_TOWN:
+		description = std::string(_("Town"));

```

Comment:
What if this was instead called "Tristram?" More specific. OTOH, it could be argued that all of these places under Tristram are also in Tristram.

Reply:
ill consider it for a follow up